### ISSUE TITLE
why can't open the website http://aix360.mybluemix.net/

### ISSUE THREAD

**Issue by ljsun** [IBM/AIX360#86]
I have tried to  open the http://aix360.mybluemix.net/ many times, but failed, so the website is not available？

**Comment by michaelhind** [comment]
Hi, I'm sorry you having trouble.  I just tried it from 2 different machines and it worked, so it maybe a local issue.

**Comment by ljsun** [comment]
Thank you, may be there is something wrong with my network

**Comment by ljsun** [comment]
None of my friends can browse the website, so I think this may be related to the region?

**Comment by rootsmusic** [comment]
Duplicate of issue #191

**Comment by rootsmusic** [comment]
I'm in the USA.  When I click on the button to "Get Started" on [The Linux Foundation's project page](https://ai-explainability-360.org/), [https://aif360.mybluemix.net/](https://aif360.mybluemix.net/) tries to load but fails.

**Comment by michaelhind** [comment]
Yes, that page is currently not reachable. We hope to have this fixed soon.
